checker/src/main/java/org/checkerframework/checker/nullness/CollectionToArrayHeuristics.java

```diff
@@ -159,6 +160,35 @@ private boolean isHandledArrayCreation(Tree argument, String receiver) {
         return false;
     }
 
+    /**
+     * Returns true if the argument is a field access expression, where the field has declared type
```

Comment:
This documentation does not seem accurate.
 * It's not any field access expression, only the special "len" field access applied to an array.
 * It's not the field that has declared type `@ArrayLen(0)`, but the array whose field is being accessed.

Reply:
There was some misunderstanding here and @xingweitian changes in response to this comment made the javadoc incorrect.
There is no special `len` field and this method didn't special-case such a field.
It is a field access where the declared type has `@ArrayLen(0)`. I'm not sure what "the array whose field is being accessed" should mean, it probably refers to the non-existent `len` field.

@xingweitian please undo your javadoc changes.